src/pybind/mgr/cephadm/module.py

```diff
@@ -2892,7 +2892,7 @@ def remove_osds(self, osd_ids: List[str],
 
         # trigger the serve loop to initiate the removal
         self._kick_serve_loop()
-        warning_zap = "" if not zap else ("\nVG/LV for the OSDs won't be zapped (--zap wasn't passed).\n"
+        warning_zap = "" if zap else ("\nVG/LV for the OSDs won't be zapped (--zap wasn't passed).\n"
                                           "Run the `ceph-volume lvm zap` command with `--destroy`"
```

Comment:
```
cephadm/module.py:2896:43: E127 continuation line over-indented for visual indent
1     E127 continuation line over-indented for visual indent
```
the quotes need to line up for flake8 I guess

Reply:
![Alt Text](https://c.tenor.com/otBqQicL6qUAAAAC/doh-homer.gif)